js/bigone.js

```diff
@@ -1228,6 +1240,84 @@ module.exports = class bigone extends Exchange {
         return this.parseTransactions (withdrawals, code, since, limit);
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const accountsById = this.safeValue (this.options, 'accountsByType', {});
+        const fromId = this.safeString (accountsById, fromAccount);
```

Comment:
Let's always support the exchange-specific identifiers for `fromAccount` and `toAccount`, that is done by adding the third argument to the safeString method which is the default value in case if the specified account is missing in `this.options['accountsByType']`. This way the user can specify either `spot` or `SPOT`, `funding` or `FUND`, etc:

```JavaScript
        const accountsById = this.safeValue (this.options, 'accountsByType', {});
        const fromId = this.safeString (accountsById, fromAccount);
        if (fromId === undefined) {
            const keys = Object.keys (accountsById);
            throw new ExchangeError (this.id + ' fromAccount must be one of ' + keys.join (', '));
        }
        const toId = this.safeString (accountsById, toAccount);
        if (toId === undefined) {
            const keys = Object.keys (accountsById);
            throw new ExchangeError (this.id + ' toAccount must be one of ' + keys.join (', '));
        }
```
↓
```JavaScript
        const accountsById = this.safeValue (this.options, 'accountsByType', {});
        const fromId = this.safeString (accountsById, fromAccount, fromAccount); // <<<<<<<< HERE
        if (fromId === undefined) {
            const keys = Object.keys (accountsById);
            throw new ExchangeError (this.id + ' fromAccount must be one of ' + keys.join (', '));
        }
        const toId = this.safeString (accountsById, toAccount, toAccount); // <<<<<<<< HEERE
        if (toId === undefined) {
            const keys = Object.keys (accountsById);
            throw new ExchangeError (this.id + ' toAccount must be one of ' + keys.join (', '));
        }
```

Reply:
Indeed, I think this was done before the same comment in the other PR. I also updated the error to be correct and I'll check the other open PRs.